bootstrap/backends/ldap/ldap.go

```diff
@@ -87,9 +90,19 @@ func NewIdentityManager(bs bootstrap.Bootstrap) (identity.Manager, error) {
 		subMapping = strings.Split(subMappingString, " ")
 	}
 
+	var tlsConfig *tls.Config
+	if caCertFile := os.Getenv("LDAP_TLS_CACERT"); caCertFile != "" {
+		var err error
+		if tlsConfig, err = utils.TlsConfigCACert(caCertFile); err != nil {
```

Comment:
This is not using the TLSConfig from config which might be confusing.

Reply:
I've reworked that. It well now alway work on a clone of  TLSConfig from config